Gemfile

```
@@ -62,6 +62,8 @@ gem "pg",                                                    :require => false
 gem "pg-dsn_parser",                    "~>0.1.0",           :require => false
 gem "psych",                            "~>3.1",             :require => false # This can be dropped once we drop ruby 2.5
 gem "query_relation",                   "~>0.1.0",           :require => false
+# rack is indirect, dependency for CVE-2024-26146 (among others)
+gem "rack",                             "~>2.2", ">= 2.2.10"
```

Minor but if you're making changes, can you inline this comment?
```suggestion

gem "rack",                             "~>2.2", ">= 2.2.10" # rack is indirect, dependency for CVE-2024-26146 (among others)
```